server/src/completion.ts

```diff
@@ -152,12 +154,15 @@ export const getCompletion =
       const command = buildCommand(config.userFlags, {
         line: position.line,
         character: column
-      }, document.languageId)
+      }, document)
+      console.log(command)
```

Comment:
Could you please remove these three unnecessary `console.log`?
(Including the one in `server/src/server.ts`)